packages/core/src/components/BaseInput/BaseInput.tsx

```diff
@@ -101,6 +105,12 @@ export interface HvBaseInputProps
   inputRef?: InputBaseProps["inputRef"];
   /** A Jss Object used to override or extend the styles applied to the component. */
   classes?: HvBaseInputClasses;
+
+  rows?: number;
+
+  minRoes?: number;
+
+  maxRows?: number;
```

Comment:
I added these because of material input type definitions, if i let these props in `others` the compiler complains of a type error because the material input component as a `or` in the type of the props (`export type InputOwnProps = (SingleLineInputProps | MultiLineInputProps)`)

Reply:
All right 👍 

If we can't enforce the generics on our side, we could use `any` to ignore it
```tsx
<MuiBaseInput
  type={type as any}
  multiline={multiline as any}
/>
```